iothub_client/CMakeLists.txt

```diff
@@ -76,23 +80,28 @@ if (use_edge_modules)
         ${CMAKE_CURRENT_LIST_DIR}/src/iothub_client_edge.c
     )
 
-    set (iothub_client_h_files
-        ${iothub_client_h_files}
+    set (iothub_client_internal_h_files
+        ${iothub_client_internal_h_files}
         ${CMAKE_CURRENT_LIST_DIR}/inc/internal/iothub_client_edge.h
     )
 endif()
 
 #this is around for back compat only
 if (${use_prov_client_core})
-    set(iothub_client_h_files
-        ${iothub_client_h_files}
+    set(iothub_client_internal_h_files
+        ${iothub_client_internal_h_files}
         ${CMAKE_CURRENT_LIST_DIR}/inc/internal/iothub_client_hsm_ll.h
     )
 endif()
 #end back compat only
 
 set(iothub_client_h_install_files
     ${iothub_client_h_files}
+    ${iothub_client_internal_h_files}
```

Comment:
Isn't `iothub_client_internal_h_files` included in `iothub_client_h_files`?

Reply:
yes, not to break the existing customers.